docs/API.md

```diff
@@ -108,7 +108,7 @@ Default: `0777`
 
 Octal permission string specifying mode for any folders that need to be created for output folder.
 
-### gulp.task(name[, deps], fn)
+### gulp.task(name [, deps] | [, deps[, cb]] | [, cb])
```

Comment:
Yeah, so which is better `gulp.task(name [, deps] [, fn])` or `gulp.task(name [, deps, fn])`? But then that means `gulp.task(name)` is valid, which isn't very useful. Also, to keep consistency, should it be `fn` or `cb` like in `gulp.watch`?